app/src/main/java/org/mozilla/fenix/settings/wallpaper/WallpaperSettings.kt

```diff
@@ -227,9 +301,10 @@ private fun WallpaperThumbnailItem(
 private fun WallpaperThumbnailsPreview() {
     FirefoxTheme(theme = Theme.getTheme()) {
         WallpaperSettings(
+            onLearnMoreClick = {},
```

Comment:
Could we reorder this to follow the order of the parameters defined by `WallpaperSettings` (eg, move it to the bottom)